app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="Log-in">
+<h2>Log in</h2>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+ <div class="field">
+    <%= f.label :username %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] I can successfully **_sign up_** (although without the confirmation being implemented) as a user in the application. However, I noticed that when I try to **_login_** with the user credentials, I get the error invalid **_username or password_**. It would be nice to fix this error 

![image](https://user-images.githubusercontent.com/77106962/226986638-b2223b81-fecf-4835-a062-c1b2e4b9f655.png)

#### Hint: Based on the requirements, you ought to log in using a combination of email and password. I would advise you to change the username form input to the email form input. That would make the login feature work. 👍 😎